pyproject.toml

```diff
@@ -24,7 +24,7 @@ dependencies = [
     "crds>=11.17.14",
     "drizzle>=1.15.0",
     "gwcs>=0.21.0,<0.22.0",
-    "numpy>=1.22",
+    "numpy>=1.22,<2.0",
```

Comment:
Setting this pin on the development branch will means we can't test with numpy 2.0. Can we set this on the release branch?